crates/oxc_minifier/src/ast_passes/peephole_substitute_alternate_syntax.rs

```diff
@@ -260,6 +267,37 @@ impl<'a> PeepholeSubstituteAlternateSyntax {
             self.changed = true;
         }
     }
+
+    fn try_compress_assignment_expression(
+        &mut self,
+        expr: &mut AssignmentExpression<'a>,
+        ctx: &mut TraverseCtx<'a>,
+    ) -> Option<Expression<'a>> {
+        let Some(target) = expr.left.as_simple_assignment_target() else { return None };
+        // TODO avoid using `clone_in`, as it is performance-intensive
+        let target = target.clone_in(ctx.ast.allocator);
```

Comment:
I think you should only clone when necessary, this is currently clone on every assignment expression?

Reply:
The clone only appears in the following situations:
- Is the target situation (from `-= 1` or `-= -1`)
- Is a simple assignment target
I am also investigating whether we can use `mem::replace`.

Reply:
I used the transform, and it is solved.